Craftgate/Response/DepositPaymentRefundResponse.cs

```diff
@@ -9,6 +9,9 @@ public class DepositPaymentRefundResponse
         public long? PaymentId { get; set; }
         public Currency Currency { get; set; }
         public RefundStatus? Status { get; set; }
+        public string AuthCode { get; set; }
```

Comment:
optional değil mi bu bilgiler failure durumunda.

Reply:
Biraz baktım. string C# da nullablemış zaten ayrıca specify etmemize gerek yokmuş.

"String is a reference type and always nullable, you don't need to do anything special. Specifying that a type is nullable is necessary only for value types."

https://stackoverflow.com/questions/32853482/how-can-i-make-my-string-property-nullable